guides/common/modules/con_upgrading-prerequisites.adoc

```diff
@@ -16,6 +16,9 @@ Upgrading {SmartProxy} takes approximately 10{range}30 minutes.
 
 * Ensure that you have sufficient storage space on your server.
 For more information, see {InstallingServerDocURL}Preparing_your_Environment_for_Installation_{project-context}[Preparing your Environment for Installation] in _{InstallingServerDocTitle}_ and {InstallingSmartProxyDocURL}preparing-environment-for-capsule-installation[Preparing your Environment for Installation] in _Installing {SmartProxyServer}_.
+* Ensure that you have at least the same amount of free space as that consumed by `/var/lib/pgsql/data` for PostgreSQL upgrade.
+The upgrade involves PostgreSQL 12 to PostgreSQL 13 upgrade.
+The existing postgreSQL data is backed up during the PostgreSQL upgrade. 
```

Comment:
At the very least, capitalization:
```suggestion
The existing PostgreSQL data is backed up during the PostgreSQL upgrade. 
```
But perhaps also use consistent terminology when referring to the data directory to help users connect the dots more easily:
```suggestion
The contents of `/var/lib/pgsql/data` is backed up during the PostgreSQL upgrade. 
```